### PR TITLE
chore: add krshkun

### PR DESCRIPTION
**Pull request by krshkun**
Adding myself

### CODE REVIEW

**Review by accodes21 — APPROVED** [review]
Looks great

File: data/users/krshkun.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Krish Gupta",
+  "bio": "Developer Advocate",
+  "image": "https://github.com/krshkun.png",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["javascript", "CSS", "react", "next", "pwa"]
```
Comment (codewithvoid) [inline data/users/krshkun.json:8]: can you limit these to max 2-3 pls.

also javascript is mentioned in programming so can drop from here

Comment (codewithvoid) [inline data/users/krshkun.json:8]: capitalise acronyms

File: data/users/krshkun.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "slug": "software-development",
+      "topics": ["git", "github", "opensource", "automation"]
+    },
+    {
+      "slug": "programming",
+      "topics": ["javascript", "node", "typescript", "discordjs", "nestjs"]
```
Comment (codewithvoid) [inline data/users/krshkun.json:16]: NodeJS and DiscordJS should go in web development category IMO

File: data/users/krshkun.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "bio": "Developer Advocate",
+  "image": "https://github.com/krshkun.png",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["javascript", "CSS", "react", "next", "pwa"]
+    },
+    {
+      "slug": "software-development",
+      "topics": ["git", "github", "opensource", "automation"]
```
Comment (codewithvoid) [inline data/users/krshkun.json:12]: max 2-3 here too

File: data/users/krshkun.json (hunk 2 above)
Comment (codewithvoid) [inline data/users/krshkun.json:16]: max 2-3 here too

**Comment by codewithvoid** [comment]
Thanks @krshkun for volunteering :)

File: data/users/krshkun.json (hunk 2 above)
Comment (krshkun) [reply data/users/krshkun.json]: Node is a server runtime, Discord is a API which also runs on the server :thinking:

Comment (codewithvoid) [reply data/users/krshkun.json]: Yes you are right. 

In programming, we are keeping programming languages and data structures and algorithms.

All server and client side frameworks and libraries are in web development.

Comment (codewithvoid) [reply data/users/krshkun.json]: I understand..its a api client library which can be used by command line applications too (not necessarily web clilents). However, like I said programming just contains languages and DSA.

You can either keep it in web development or add it later once we have category for CLI tools.

File: data/users/krshkun.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      "slug": "web-development",
+      "topics": ["CSS", "react", "next"]
+    },
+    {
+      "slug": "software-development",
+      "topics": ["git", "github", "opensource"]
+    },
+    {
+      "slug": "programming",
+      "topics": ["javascript", "typescript", "discordjs"]
```
Comment (krshkun) [inline data/users/krshkun.json:16]:
```suggestion
      "topics": ["javascript", "typescript"]
```

**Comment by codewithvoid** [comment]
Perfect. Welcome Krish ser!

**Pull request merged**